libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -260,11 +260,14 @@ void AP_MotorsHeli_Dual::set_desired_rotor_speed(float desired_speed)
 // calculate_armed_scalars
 void AP_MotorsHeli_Dual::calculate_armed_scalars()
 {
+    float thrcrv[5];
+    int i;
```

Comment:
another spot to change i to uint8_t (also put the loop variable inside the for loop, like this:
for (uint8_t i=0; i<5; i++) 
that minimizes its scope